willow/plugins/wand.py

```diff
@@ -230,6 +231,7 @@ def get_wand_image(self):
     @Image.converter_from(TIFFImageFile, cost=150)
     @Image.converter_from(WebPImageFile, cost=150)
     @Image.converter_from(HeicImageFile, cost=150)
+    @Image.converter_from(AvifImageFile, cost=150)
     def open(cls, image_file):
```

Comment:
we're missing a `save_as_avif` image operation here. Imagemagick supports AVIF via libheic. and out wand plugin does have a save_as_webp operation